packages/image_picker/image_picker/test/image_picker_test.dart

```diff
@@ -315,6 +315,41 @@ void main() {
           expect(response.file!.path, '/example/path');
         });
 
+        test('retrieveLostData get success response with pathList', () async {
+          channel.setMockMethodCallHandler((MethodCall methodCall) async {
+            return <String, dynamic>{
+              'type': 'image',
+              'path': '/example/path',
+              'pathList': ['/example/path'],
+            };
+          });
+
+          final LostDataResponse response = await picker.retrieveLostData();
+          expect(response.type, RetrieveType.image);
+          expect(response.file, isNotNull);
+          expect(response.file!.path, '/example/path');
+          expect(response.files!.first.path, '/example/path');
+        });
+
+        test('retrieveLostData is failed for multiRetrieve method call',
```

Comment:
I'd change this to something a bit more descriptive like:
```suggestion
        test('retrieveLostData returns null for files property on MissingPluginException',
```
(Assuming that's a correct description)